src/js/Tables/BaseTable.js

```diff
@@ -142,11 +142,52 @@ class BaseTable {
 	}
 
 	/**
-	 * Update table rows.
+	 * Update the o-table instance with rows added or removed dynamically from
+	 * the table. Applies any existing sort and filter to new rows.
 	 *
 	 * @returns {undefined}
 	 */
 	updateRows() {
+		const rows = this._getLatestRowNodes();
+		// Find rows added.
+		const newRows = rows.filter(function (row) {
+			return this.tableRows.indexOf(row) === -1;
+		}.bind(this));
```

Comment:
instead of doing `.bind(this)` you can pass `this` as the second parameter to `rows.filter`

Reply:
fancy pants

Reply:
done